File: twiml/voice/conference/conference-10/conference-10.5.x.rb
require 'twilio-ruby'

response = Twilio::TwiML::VoiceResponse.new
response.dial do |dial|
  dial.conference('LoveTwilio', record: 'record-from-start',
                                recording_status_callback: 'www.myexample.com')
end

puts response

File: twiml/voice/conference/conference-10/conference-10.5.x.js
const VoiceResponse = require('twilio').twiml.VoiceResponse;


const response = new VoiceResponse();
const dial = response.dial();
dial.conference({
    record: 'record-from-start',
    recordingStatusCallback: 'www.myexample.com'
}, 'LoveTwilio');

console.log(response.toString());
